### ISSUE TITLE
Can't update license.

### ISSUE THREAD

**Issue by supportnubix** [mageplaza/magento-2-smtp#216]
Hello,

Using your module on Magento 2.2.8 on version 1.2.10 running on PHP 7.1 we cannot update the license.

We ran into the issue because on the development website we tried sending e-mails but the module refused to send it. On live it is working perfectly.

On development we tried to use login information blabla@blabla.com with random password and even then it said e-mail was sent succesfully, that doesn't seem possible.

We did not update in between, we did not change anything in between and it just stopped working.

So we got 2 issues.

1. Can't update license, you change information click on submit and the "Save" button greys out, if refreshed then old information is back.
2. It says e-mail sent succefully even though it can never be sent succesfully with fake information filled in into the login.

See attachment.
![image](https://user-images.githubusercontent.com/32568309/71086529-8055ac80-219a-11ea-9e4d-03220759135f.png)

What can we do?

**Comment by Arthas7895** [comment]
About the update license problem, I can confirm that it does not affect the SMTP information as well as the mail flow on your site. You should try with email addresses with common domain names like Gmail
For the other issue, make sure you have configured the following fields correctly || https://prnt.sc/qcuh29 || https://prnt.sc/qcuh7p

**Comment by supportnubix** [comment]
Hello @Arthas7895 thanks for responding.
Good to know updating the license doesn't have anything to do with it.

Sadly, everything seems to be configured the same as live environment, on staging it also was working before but at some (unknown) point is stopped working, we set up a second enviroment based on live and the "issue" still occurs.

Developer mode is off, I've tested all kinds of options with real and fake adresses and providers ( the fake adresses should simply fail )

No idea where to look from here.
